bundles/org.openhab.core/src/main/java/org/openhab/core/library/types/QuantityType.java

```diff
@@ -97,7 +98,13 @@ public QuantityType(String value) {
 
         // getQuantity needs a space between numeric value and unit
         String formatted = String.join(" ", constituents);
-        quantity = (Quantity<T>) Quantities.getQuantity(formatted);
+        if (!formatted.contains(" ")) {
+            BigDecimal bd = new BigDecimal(value);
+            quantity = (Quantity<T>) Quantities.getQuantity(bd, AbstractUnit.ONE, Scale.RELATIVE);
```

Comment:
@wborn, do you remember why you picked `Scale.RELATIVE` here? While reviewing #3177 I found that the default is `Scale.ABSOLUTE` but we explicitly set it to a different value. Instinctively I would have chosen `Scale.ABSOLUTE` because that is what we usually get when we talk e.g. about temperatures.

Reply:
IIRC such a change would cause a lot more rules to break.

Reply:
I didn't want to suggest changing it. I was just wondering if this was a deliberate decision or not. Because #3177 only works because of this.

Reply:
If there was no legacy code and there were no openHAB users I would also opt for using all the default behaviors. :slightly_smiling_face: